src/seedu/addressbook/commands/EditCommand.java

```diff
@@ -0,0 +1,50 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.common.Messages;
+import seedu.addressbook.data.exception.IllegalValueException;
+import seedu.addressbook.data.person.Person;
+import seedu.addressbook.data.person.UniquePersonList.PersonNotFoundException;
+
+/**
+ * Edits a specified person's detail identified using the last displayed index from the address book.
+ */
+public class EditCommand extends Command{
```

Comment:
There should be a whitespace before the braces {. Minor detail but stick to coding standards.